### ISSUE TITLE
Fix various  issues with "meta" docs in `projects/core/howto`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4949]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4949|
|Type|enhancement|
|Created|2011-03-15 01:04:46Z|

various issues:

 * **debug-with-emacs**: pokey is missing, reinstate him

 * **glossary**: Consider is the removal of the (currently broken) link from the "Spread" entry title: We could keep it as a link to core/howto/pb.html, which is titled "Overview Of Twisted Spread" and seems like a good place to send readers who want to learn more about Spread.

 * **upgrading**: Looks good, but I noticed that under "Rebuild: Loading New Code Without Restarting", the API link is being removed from the first mention of Versioned (seemingly because it was broken?) But the second appearance of Versioned in that section has a working link to [http://twistedmatrix.com/documents/current/api/twisted.persisted.styles.Versioned.html]; It seems like the first mention should link to the same place. Later mentions of both rebuild and Versioned aren't links, but that seems to be on purpose. (I'm guessing the pattern is that the first mention in a paragraph is a link and the rest aren't?)

Attachments:

* [versioned.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5fb/5fbc756c6c029ad667640367e835fb24d9a0bfc1/b067cab18efa62f0b5d37766bba156c96d266195.patch) (992 bytes) - added by thijs on 2011-07-01 16:09:57Z - 
* [versioned.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5fb/5fbc756c6c029ad667640367e835fb24d9a0bfc1/eddd94fd7849e5bff338ec0f5e0e4c0457556640.patch) (2734 bytes) - added by thijs on 2011-07-05 06:44:41Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4949 4949
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1300151086000000 1300151086000000
changetime__1309848455000000 1309848455000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Replying to [khorn](#4949):
> various issues:
> 
>  * **debug-with-emacs**: pokey is missing, reinstate him

I can see the image in my local version as well as [online](http://twistedmatrix.com/documents/current/core/howto/debug-with-emacs.html)?

> 
>  * **glossary**: Consider is the removal of the (currently broken) link from the "Spread" entry title: We could keep it as a link to core/howto/pb.html, which is titled "Overview Of Twisted Spread" and seems like a good place to send readers who want to learn more about Spread.

Looks like this was fixed in r31099.

> 
>  * **upgrading**: Looks good, but I noticed that under "Rebuild: Loading New Code Without Restarting", the API link is being removed from the first mention of Versioned (seemingly because it was broken?) But the second appearance of Versioned in that section has a working link to [http://twistedmatrix.com/documents/current/api/twisted.persisted.styles.Versioned.html]; It seems like the first mention should link to the same place. Later mentions of both rebuild and Versioned aren't links, but that seems to be on purpose. (I'm guessing the pattern is that the first mention in a paragraph is a link and the rest aren't?)

Fixed in attached patch.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for all your work on these documentation tickets, thijs.

Your changes look good to ship. While you're in that file, it'd be great if you could correct a few spelling errors:

 * accomodate ==> accommodate
 * compatiblity ==> compatibility
 * compability ==> compatibility

Other than that, looks great, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32228]) Apply versioned.2.patch: Add additional link for t.persisted.styles.Versioned and correct some typos.

Author: thijs
Reviewer: jesstess
Fixes: [#4949](https://github.com/twisted/twisted/issues/4949)
